docs/supported_apis/series_supported.rst

```diff
@@ -13,473 +13,480 @@ for partial (meaning some parameters may not be supported yet), and ``D`` stands
-| ``to_sql``                  | Y                               |
-+-----------------------------+---------------------------------+
-| ``to_string``               | D                               |
-+-----------------------------+---------------------------------+
-| ``to_timestamp``            | D                               |
-+-----------------------------+---------------------------------+
-| ``to_xarray``               | D                               |
-+-----------------------------+---------------------------------+
-| ``tolist``                  | D                               |
-+-----------------------------+---------------------------------+
-| ``transform``               | Y                               |
-+-----------------------------+---------------------------------+
-| ``transpose``               | Y                               |
-+-----------------------------+---------------------------------+
-| ``truediv``                 | Y                               |
-+-----------------------------+---------------------------------+
-| ``truncate``                | Y                               |
-+-----------------------------+---------------------------------+
-| ``tshift``                  | Y                               |
-+-----------------------------+---------------------------------+
-| ``tz_convert``              | Y                               |
-+-----------------------------+---------------------------------+
-| ``tz_localize``             | Y                               |
-+-----------------------------+---------------------------------+
-| ``unique``                  | Y                               |
-+-----------------------------+---------------------------------+
-| ``unstack``                 | Y                               |
-+-----------------------------+---------------------------------+
-| ``update``                  | Y                               |
-+-----------------------------+---------------------------------+
-| ``valid``                   | D                               |
-+-----------------------------+---------------------------------+
-| ``value_counts``            | Y                               |
-+-----------------------------+---------------------------------+
-| ``values``                  | Y                               |
-+-----------------------------+---------------------------------+
-| ``var``                     | Y                               |
-+-----------------------------+---------------------------------+
-| ``view``                    | D                               |
-+-----------------------------+---------------------------------+
-| ``where``                   | Y                               |
-+-----------------------------+---------------------------------+
++-----------------------------+---------------------------------+----------------------------------------------------+

```

Comment:
When I open this file via `view file`, the table is not displayed. I think, you edited file a little incorrectly. Please, fix this and check other doc files in order to correct displaying of the tables.